cli/config/configuration.toml.in

```diff
@@ -75,7 +75,7 @@ default_bridges = @DEFBRIDGES@
 
 # Default memory size in MiB for SB/VM.
 # If unspecified then it will be set @DEFMEMSZ@ MiB.
-#default_memory = @DEFMEMSZ@
+default_memory = @DEFMEMSZ@
```

Comment:
Is its behavior changed? Not a big problem.

Reply:
I change this because `KataConfig.Hypervisor[DefaultHypervisor].DefaultMemSz` in https://github.com/kata-containers/tests/pull/813 need the value of `default_memory` in toml file.

See https://github.com/kata-containers/tests/blob/master/config.go#L37

Reply:
This might need to be release-noted then as users with a `/etc/kata-containers/configuration.toml` file probably *won't* have that option enabled since it's likely to be based on the current config file we distribute as `/usr/share/defaults/kata-containers/configuration.toml`.

/cc @egernst, @jcvenegas, @bergwolf.

Reply:
As I change cli/config/configuration.toml.in just for CI testcase, which I think will not affect users. So maybe there is no need to release-noted for now as long as the CI won't fail.